spec/lib/hyrax/configuration_spec.rb

```diff
@@ -55,4 +55,5 @@
   it { is_expected.to respond_to(:translate_uri_to_id) }
   it { is_expected.to respond_to(:upload_path) }
   it { is_expected.to respond_to(:work_requires_files?) }
+  it { is_expected.to respond_to(:extract_full_text?) }
 end
```

Comment:
Can we also check that the default is actually `true`, both when the config value is explicitly set and when it is not?